litmus-portal/frontend/src/pages/LoginPage/index.tsx

```diff
@@ -1,62 +1,113 @@
-import { Typography, Paper, TextField, Hidden } from '@material-ui/core';
+import { Typography, TextField, Hidden, Button } from '@material-ui/core';
 import { Link } from 'react-router-dom';
-import React from 'react';
-import ButtonFilled from '../../components/ButtonFilled/index';
+import React, { useState } from 'react';
 import useStyles from './styles';
 
+interface authData {
+  username: string;
+  password: string;
+}
+
 const LoginPage = () => {
   const classes = useStyles();
+  const [authData, setAuthData] = useState<authData>({
+    username: '',
+    password: '',
+  });
+  const handleForm = () => {
+    const formData: HTMLFormElement | null = document.querySelector(
+      '#login-form'
+    );
+    const data = new FormData(formData as HTMLFormElement);
+    const username = data.get('username') as string;
+    const password = data.get('password') as string;
+    data.append('username', username);
+    data.append('password', password);
+    fetch('/login', {
+      method: 'POST',
+      body: data,
+    }).then((response) => {
+      response.text();
+    });
+  };
   return (
```

Comment:
Add one line before the return statement

Reply:
Added the changes @rajdas98 PTAL